main.js

```diff
@@ -0,0 +1,82 @@
+// #1
+var sum = items.reduce(function (a, b) {
+	return a + b.price;
+}, 0)
+
+var avg = sum/items.length;
+
+document.querySelector("#answer1").innerHTML = "The average price is $" + avg.toFixed(2);
+
+// #2
+var priceMatch = items.filter(function (item) {
+	if (item.price > 14 && item.price < 18) {
+		return "item.title;"
```

Comment:
One note here is that you are returning a string not an actual value.  The only reason this works is because javascript evaluates a string as true